source/includes/fact-verifier-memory.rst

```diff
@@ -0,0 +1,15 @@
+
+Verification requires 10 GB of memory plus an additional 500 MB
+for every 1 million documents in the migration. 
+
+- If the server ``mongosync`` runs on doesn't have sufficient
+  memory available, the ``/start`` endpoint returns an error. 
+
+- If another process claims this memory, the verifier can cause
+  the server to stop the ``mongosync`` process due to an out of
+  memory (OOM) error.
```

Comment:
So sorry to go back and forth on this - let's remove this OOM note on second thought. We can add it back if users do end up running into this, but this is expected to be unlikely

Reply:
Fixed